libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/parametersTab/index.tsx

```diff
@@ -89,6 +94,7 @@ export const ParametersTab: React.FC<PanelTabProps> = (props) => {
   const errorInfo = useOperationErrorInfo(selectedNodeId);
   const { hideUTFExpressions } = useHostOptions();
   const replacedIds = useReplacedIds();
+  const parentIdOfSwitch = useGetSwitchParentId(selectedNodeId);
```

Comment:
We already have this info in the NodeMetadata in redux, on line 80 `nodeMetadata.parentNodeId`